libs/application-generic/src/usecases/create-notification-jobs/create-notification-jobs.usecase.ts

```diff
@@ -64,6 +64,7 @@ export class CreateNotificationJobs {
       channels,
       controls: command.controls,
       tags: command.template.tags,
+      preferences: command.template.preferences,
```

Comment:
Taking the same approach as we had with `tags`, by injecting the `preferences` onto the mapped `template`.